src/clj/harja/kyselyt/sopimukset.sql

```diff
@@ -75,6 +75,8 @@ FROM sopimus s
   LEFT JOIN urakka u ON s.urakka = u.id
   LEFT JOIN kan_lt_ketjutus_aktivointi k ON k.sopimus_id = s.id
 WHERE u.tyyppi = 'vesivayla-kanavien-hoito' 
+AND ( :urakka-id < 0 OR (:urakka-id = u.id) ) 
```

Comment:
Oliko tähän tarkoitus kirjoittaa :urakka-id IS NULL OR (:urakka-id = u.id)? Tällainen on ainakin ollut yleensä tyylinä näissä.

Reply:
https://github.com/finnishtransportagency/harja/pull/2886/commits/3ee6a1e64904401ff9cd5bceb31f5fb6f7c4f0dc